cmd/controller-manager/app/options/options.go

```diff
@@ -74,6 +74,8 @@ type Options struct {
 	// ResyncPeriod is the base frequency the informers are resynced.
 	// Defaults to 0, which means the created informer will never do resyncs.
 	ResyncPeriod metav1.Duration
+	// MetricsAddr is address the metric endpoint binds to
+	MetricsAddr string
```

Comment:
```suggestion

	// MetricsBindAddress is the TCP address that the controller should bind to
	// for serving prometheus metrics.
	// It can be set to "0" to disable the metrics serving.
	// Defaults to ":8080".
	MetricsBindAddress string
```